### DISCUSSION TITLE
IPv6 support for Vercel's Edge Network

### DISCUSSION THREAD

**Discussion by TomsProject** [vercel/community#1347] [Help] [answered]
IPv6 support for Vercel's Edge Network
AWS Global Accelerator supports IPv6 now, so I hope Vercel's Edge Network could support IPv6 too.

https://aws.amazon.com/cn/blogs/networking-and-content-delivery/introducing-aws-global-accelerator-ipv6/

**Comment by amyegan** [comment, marked as answer]
We are working toward this, but there is a technical limitation that needs to be addressed before our Edge Network can support IPv6. If you have strict requirements for IPv6, [please let us know](https://vercel.com/contact/sales) and we can discuss potential solutions in depth.
